app.js

```diff
@@ -227,6 +227,23 @@ function App (el) {
     })
   })
 
+  self.on('executeCommand', function (commandStr) {
+    var words = commandStr.split(' ')
+    var command = words[0].substring(1, words[0].length).toLowerCase()
+    var message
+
+    switch (command) {
+      case 'join':
+        words.shift()
+        var channel = words.join(' ')
```

Comment:
you mean `/join `? good q

maybe we need one more event for handle different type of commands. i.e just `/part` command and `/join (#)channel`